src/main/java/forum/dao/ReplyDAOImpl.java

```diff
@@ -0,0 +1,36 @@
+package forum.dao;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import forum.entity.Reply;
+
+@Repository
+public class ReplyDAOImpl implements ReplyDAO {
+	
+	@Autowired
+	private SessionFactory sessionFactory;
+
+	@Override
+	public int saveReply(Reply reply) {
+		
+		Session currentSession = sessionFactory.getCurrentSession();
+		
+		int result = (int) currentSession.save(reply);
```

Comment:
Can this variable name be more specific? Like `replyId` etc :D

Reply:
Yes, you are right. I renamed the variable..